frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushAction.java

```diff
@@ -66,16 +66,16 @@ protected void onUpdate(AnActionEvent anActionEvent) {
     syncToRemoteStatusDependentActionUpdate(anActionEvent);
 
     val branchName = getNameOfBranchUnderAction(anActionEvent);
```

Comment:
This error with `lombok.val` (`BasePushAction.java:68: error: incompatible types: @org.checkerframework.checker.nullness.qual.Nullable String cannot be converted to val`) can be just solved with changing to explicit type (`String`) or `var`